articles/howto-build-astatic-site-with-gatsbyjs/index.md

```diff
@@ -0,0 +1,281 @@
+# How to build a static site with GatsbyJs
+![Hero image](/engineering-education/howto-build-astatic-site-with-gatsbyjs/heroimage.png)
+[*Image Source: Unsplash](https://unsplash.com/photos/uyfohHiTxho)
+
+When I first heard about GatsbyJS, I was confused about what it was, and how I would get started. As time went on, I learned more about GatsbyJS and found out it is awesome. My goal for this article is not to confuse you about GatsbyJS, but to help you get started with building your very first static site using GatsbyJS.
+
+GatsbyJS is used to build websites powered by [JAMStack](https://jamstack.org/what-is-jamstack/). JAMStack is a modern framework for creating websites and software. The “JAM” is an acronym for JavaScript, APIs, and HTML markup, JAMStack sites do not need a database, unlike websites designed using WordPress or Drupal. 
+In JAMStack architecture, the frontend and backend are [decoupled](https://www.gatsbyjs.com/docs/glossary/#decoupled). During the building process, Gatsby creates JavaScript, HTML, and CSS files used for the frontend, while the backend is a content API that can be a headless [CMS](https://en.wikipedia.org/wiki/Content_management_system), a hosted datastore, or a custom application that returns JSON or XML.
```

Comment:
- This is a load of jargon. I see that you included a link, which is great, but you have to remember that the audience is a beginner. It may benefit them to have a brief explanatory sentence of two to avoid opening too many external links and be able to follow along with the rest of the article.

Reply:
- In addition, what is "headless" CMS??
- What does decoupled mean? Reader can't follow.

Reply:
- Content API?

Reply:
Done